enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,346 @@
+---
+title: automated-backups-no-config
+authors:
+  - "@elbehery"
+reviewers:
+  - "@JoelSpeed"
+  - "@dusk125"
+  - "@hasbro17"
+  - "@tjungblu"
+  - "@vrutkovs"
+  - "@jmhbnz"
+  - "@deads2k"
+  - "@csrwng"
+  - "@sjenning"
+  - "@wking"
+  - "@rhuss"
+  - "@gnufied"
+approvers:
+  - "@deads2k"
+  - "@JoelSpeed"
+  - "@hasbro17"
+  - "@tjungblu"
+  - "@vrutkovs"
+  - "@JoelSpeed"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2024-06-17
+last-updated: 2024-10-21
+tracking-link:
+  - https://issues.redhat.com/browse/ETCD-609
+see-also:
+  - "https://issues.redhat.com/browse/OCPSTRAT-1411"
+  - "https://issues.redhat.com/browse/OCPSTRAT-529"
+---
+
+
+# Automated Backups of etcd with No Config
+
+## Summary
+
+To enable automated backups of Etcd database and cluster resources of an Openshift cluster from day one after installation, 
+without provided configuration from the user.
+
+This document outlines the possible approaches that were investigated, discussed and tested, in addition to the final implemented approach.  
+
+## Motivation
+
+The current method of **Backup** and **Restore** of an Openshift cluster is outlined in  [Automated Backups of etcd](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md). 
+This approach relies on user supplied configuration, in the form of `config.openshift.io/v1alpha1 Backup` CR. This method utilizes `batchv1.CronJob` in order to take Backups of Etcd and cluster resources.
+
+To improve the user's experience, this work proposes taking etcd backup using default configurations from day one, without user intervention.
+
+Moreover, for recovery in disaster scenarios, where more than one control-plane node is lost from an Openshift cluster, this proposal attempts to store backups on all control-plane nodes.
+Having backups of an Openshift cluster on all control-plane nodes, improves the recovery probability in disaster scenarios.
+
+### User Stories
+
+* As a cluster administrator, I would like an OCP cluster backups to be taken without user's supplied configuration from day one, after installation.
+* As a cluster administrator, I would like to an OCP cluster backups available on all master nodes, so that I have a recent cluster state to recover from in the event of losing control-plane nodes.
+
+### Goals
+
+* Backups should be taken without configuration after cluster installation from day 1.
+* Backups are saved to a default location on each master node's file system.
+* Backups are taken according to a default schedule.
+* Backups are being maintained according to a default retention policy.
```

Comment:
Would it also be a goal to ensure that the backup is valid? Is there a periodic job that ensures this or is the cluster/customer trusting it will work?

Reply:
I believe the size of the etcd db is no longer capped. Is this a concern for taking snapshots?